src/ecs/resources/input/input_handler.rs

```diff
@@ -267,40 +215,42 @@ impl InputHandler {
 
     /// Checks if all the given mouse buttons are being pressed.
     pub fn mouse_buttons_are_pressed(&self, buttons: &[MouseButton]) -> bool {
```

Comment:
Why do you need to check if several mouse buttons were pressed last frame? It may be really hard to do!
Maybe you need to check if they all down simultaneously?